spec/test_cbw_api.py

```diff
@@ -2,7 +2,6 @@
 
 from cbw_api_toolbox.cbw_api import CBWApi
 from cbw_api_toolbox.cbw_objects.cbw_server import CBWServer
-from cbw_api_toolbox.cbw_objects.cbw_remote_access import CBWRemoteAccess
```

Comment:
why?

Reply:
Because it's not used by the update_remote_access test